hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/PhoneHome.java

```diff
@@ -50,21 +51,31 @@
     volatile ScheduledFuture<?> phoneHomeFuture;
     private final ILogger logger;
     private final String basePhoneHomeUrl;
+    private final HashMap<String, String> envVars;
     private final List<MetricsCollector> metricsCollectorList;
 
     public PhoneHome(Node node) {
         this(node, DEFAULT_BASE_PHONE_HOME_URL, CLOUD_INFO_COLLECTOR);
     }
 
-    @SuppressWarnings("checkstyle:magicnumber")
+    PhoneHome(Node node, String basePhoneHomeUrl, Map<String, String> envVars) {
+        this(node, basePhoneHomeUrl, envVars, CLOUD_INFO_COLLECTOR);
```

Comment:
nit: the same param has different names in constructors `basePhoneHomeUrl` and  `baseUrl`

Reply:
fixed